### ISSUE TITLE
We need a way to serialize a Policy

### ISSUE THREAD

**Issue by tuxxy** [nucypher/nucypher#741]
In our Alice's Character Control, users will be able to create a policy via `create_policy`. When the policy is created, it needs to be serialized over the wire back to the requestor.

**Comment by drbh** [comment]
I agree! In my dapp I also need to include the IPFS CID of the file. 

I decided to go with this format  `ncipfs://CID_POLICYID_SIGID_LABEL`

example:  
`ncipfs://QmWzx5JKSjnBdnR8EhRPNyXQka2KZJiiXKu2jPqT7xHjQT_uxLpz8qvdVSB4yjpL6gX4LgYHzWhWxrYnmKQ3cyHHSic_xc8UZXUtf5APKX4ZsjXJnFJVJaL4XKXNMvuagVKCtWBt_image.png`

I encoded the policy and sig id with `base58` since it does not include word-breaking chars. This way a user can double click any of the values and the whole continuous sting will be selected. 

snippet:  
```python
import base58
enc_str = base58.b58encode(enc_pubkey.to_bytes()).decode("utf-8")
sig_str = base58.b58encode(sig_pubkey.to_bytes()).decode("utf-8")
```

Lastly I prefixed the string with `ncipfs://` a IRI look-a-like prefix for my dapp to understand how to handle the data. 

This encoding seems sufficient for my dapp's purposes but I would like to follow best practices. Are there ways we can shorten this identifier as well?
